x/oracle/types/currency_pair.go

```diff
@@ -6,14 +6,16 @@ import (
 )
 
 const (
-	ethereum = "ETHEREUM"
+	DefaultDecimals  = 8
+	EthereumDecimals = 18
 )
 
 // NewCurrencyPair returns a new CurrencyPair with the given base and quote strings.
-func NewCurrencyPair(base, quote string) CurrencyPair {
+func NewCurrencyPair(base, quote string, decimals int64) CurrencyPair {
```

Comment:
Yes!

one question I had is that, since we are doing this, should we let people query using both 

`BASE/QUOTE/DEC`
and
`BASE/QUOTE` 

? 

IMO i think simplifying to just use `BASE/QUOTE` for the query would be good, but down to discuss.